apps/01PigLatin.js

```diff
@@ -19,29 +19,29 @@ function pigLatin(word) {
   var vowelIndex = -1; // Set it to assume there are no vowels
 
   if ((word.indexOf('a') > -1 && word.indexOf('a') < vowelIndex) || vowelIndex === -1) {
-      vowelIndex = word.indexOf('a');
```

Comment:
Omg, I had trouble with these indentions, too! Did you end up setting the correct indentions in your Beautify?